tests/Integration/Endpoints/RecommendationsProxyEndpointTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Parsely Recommendations API Proxy Endpoint tests.
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration;
+
+use Parsely\Parsely;
+use Parsely\Endpoints\Recommendations_API_Proxy;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Server;
+
+/**
+ * Parsely REST API tests.
+ */
+final class RecommendationsProxyEndpointTest extends TestCase {
+	/**
+	 * Set up globals & initialize the Endpoint.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		add_filter( 'wp_parsely_enable_recommendations_endpoint', '__return_true' );
+		update_option( 'parsely', array( 'apikey' => 'example.com' ) );
+
+		$this->wp_rest_server_global_backup        = $GLOBALS['wp_rest_server'] ?? null;
+		$this->rest_api_init_recommendations_proxy = function () {
+			$endpoint = new Recommendations_API_Proxy( new Parsely() );
+			$endpoint->run();
+		};
+		add_action( 'rest_api_init', $this->rest_api_init_recommendations_proxy );
+	}
+
+	/**
+	 * Reset globals.
+	 */
+	public function tearDown(): void {
+		parent::tearDown();
+		remove_action( 'rest_api_init', $this->rest_api_init_recommendations_proxy );
+		$GLOBALS['parsely'] = $this->parsely_global_backup;
+		// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
+		$GLOBALS['wp_rest_server'] = $this->wp_rest_server_global_backup;
+		delete_option( 'parsely' );
+		remove_filter( 'wp_parsely_enable_recommendations_endpoint', '__return_true' );
+	}
+
+	/**
+	 * Confirm the the route is registered.
+	 *
+	 * @covers Recommendations_API_Proxy::register_rest_route
+	 */
+	public function test_register_routes() {
+		$routes = rest_get_server()->get_routes();
+		$this->assertArrayHasKey( '/wp-parsely/v1/recommendations', $routes );
+		$this->assertSame( 1, count( $routes['/wp-parsely/v1/recommendations'] ) );
```

Comment:
Use `assertCount()` instead.

```suggestion
		$this->assertCount( 1, $routes['/wp-parsely/v1/recommendations'] );
```